README.md

```diff
@@ -1,7 +1,7 @@
 # bridge-filtering
 
 This CNI plugin allows the user to control traffic flow at the IP address or port level (OSI layer 3 or 4) for particular applications in the Kubernetes cluster, thus specifying how a pod is allowed to communicate with various network "entities".
-The supported L4 protocols are UDP and TCP.
+The supported L4 protocols are UDP and TCP. All ICMP traffic is allowed by `bridge-filtering-cni`, for ways to control ICMP traffic, see your CNI documentation.
```

Comment:
I would  drop the second part, there may be no notion of ICMP in their docs, so this may be misleading

```suggestion
The supported L4 protocols are UDP and TCP. All ICMP traffic is allowed by `bridge-filtering-cni`.
```

Reply:
Ah, that's a good point